immutable.js

```diff
@@ -35,6 +42,11 @@ console.log(combinedSupers);
   Create a new object `studentAverage` using Object.assign() from the given original object.
   The new object should filter out any scores that are < 34 Array.filter should be used here.
 */
+const studentScores = { name: "jim", age: "1", scores: [1, 10, 35, 50] };
+const studentAverage = Object.assign({}, studentScores);
+studentAverage.scores = studentAverage.scores.filter(score => {
```

Comment:
If immediately returning out of an arrow function jut remove the brackets and skip out on the return.
```
studentAverage.scores = 
  studentAverage
    .scores
    .filter(
      score =>
        score >= 34
    );
```
This is just nitpicking but something like this will increase code readability imo.